### PR TITLE
BZ1793578 - Add OCS 4.2 to OCP 4.2 support types

### PR DESCRIPTION
**Pull request by bobfuru**
https://bugzilla.redhat.com/show_bug.cgi?id=1793578
@liangxia PTAL

### CODE REVIEW

**Comment by liangxia** [comment]
/lgtm
Shall we add it to OCP 4.3 too ?

File: modules/storage-persistent-storage-pv.adoc [hunk 1]
```diff
@@ -145,6 +145,7 @@ the Pods that use these volumes are deleted.
 |iSCSI  | ✅ | ✅ |  -
 |Local volume | ✅ | - |  -
 |NFS  | ✅ | ✅ | ✅
+|Red Hat OpenShift Container Storage | ✅ | ✅ | ✅
```
Comment (liangxia) [inline modules/storage-persistent-storage-pv.adoc]: Need OCS QE to confirm that it support access mode RWO,ROX,RWX

Comment (bobfuru) [reply modules/storage-persistent-storage-pv.adoc]: @ebenahar or @RazTamir Could one of you PTAL and confirm with `LGTM`? Thanks.

Comment (RazTamir) [reply modules/storage-persistent-storage-pv.adoc]: ROX is not supported

Comment (bobfuru) [reply modules/storage-persistent-storage-pv.adoc]: @RazTamir Removed ROX. Look good?

**Review by RazTamir — APPROVED** [review]
LGTM

File: modules/storage-persistent-storage-pv.adoc (hunk 1 above)
Comment (RazTamir) [reply modules/storage-persistent-storage-pv.adoc]: Yes, thank you.
LGTM

**Comment by bobfuru** [comment]
@openshift/team-documentation PTAL

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**